cal-haskell.cabal

```diff
@@ -19,7 +19,7 @@ synopsis:            Trying to learn Haskell for great good!
 -- description:
 
 -- URL for the project homepage or repository.
-homepage:            git@github.com:erlandsona/cal_haskell.git
+homepage:            https://github.com/erlandsona/cal_haskell
```

Comment:
I added a separate `source-repository` section below that covers the git specifics. This field is typically used for a HTTP page.